core/src/test/java/com/novoda/merlin/service/PingShould.java

```diff
@@ -23,66 +21,54 @@
 
     private static final String HOST_ADDRESS = "any host address";
 
-    @RunWith(Parameterized.class)
     public static class GivenSuccessfulRequest {
 
-        @Parameterized.Parameters(name = "{0}")
-        public static Collection<Integer[]> data() {
-            return ResponseCode.toParameterList();
-        }
-
-        private final Ping ping;
+        private Ping ping;
 
         @Mock
-        private HostPinger.ResponseCodeFetcher mockResponseCodeFetcher;
+        private ResponseCodeFetcher mockResponseCodeFetcher;
+        @Mock
+        private ResponseCodeValidator mockResponseCodeValidator;
 
-        private final int responseCode;
+        private final int responseCode = 234;
```

Comment:
I liked having it as a fake response code because it helps to understand the actual value doesn't matter at all. `fakeResponseCode` or `anyResponseCode` could be better though. Will update